src/js/components/Data/stories/Inline.js

```diff
@@ -0,0 +1,97 @@
+import React, { useContext } from 'react';
```

Comment:
After applying the filter 'small' on this story, I can no longer see the medium filter. I would expect that both options would still show up in the filters list.

https://user-images.githubusercontent.com/54560994/206579221-f82cb165-ad3c-4ae9-a6ac-eeefbcf3c9f3.mp4

Reply:
Excellent catch! Fixed.